### PR TITLE
Updated data with headline figures and appropriate disaggregation

### PR DESCRIPTION
**Pull request by AlexLewis1**
This pull request has been automatically generated by prose.io.

### CODE REVIEW

**Comment by AlexLewis1** [comment]
I have deleted the spaces in the first and last column headers but it still seems to be failing. Where can I view where the error is occurring?

**Comment by dougmet** [comment]
Hi Alex, thanks for your patience! 

This error actually looks like it's my problem. The build thinks you have non-numeric values in the "Value" column, but I can see that you don't, so I'll take a look right away.

In general for debugging problems it is a little tricky fishing out what went wrong. If you click on the "Details" link next to the build you can see these lines:

```
$ ruby ./scripts/csvcheck.rb
All entries in Value must be missing or numeric: data/indicator_3-2-2.csv
```

I appreciate that this is not easy to find! I'm going to investigate how hard it would be to have travis post the error back to a pull request to make this all easier.

**Comment by AlexLewis1** [comment]
Hi Doug,

Thanks a lot for looking into this. I did look into the details section initially but nothing jumped out at me, but I'll keep in mind for the moment that the errors will take a bit of digging to find!

**Comment by dougmet** [comment]
If it helps, in the short term, there will be red text for the validation error. The text above that is where it tells you the problem:

```
$ ruby ./scripts/csvcheck.rb
All entries in Value must be missing or numeric: data/indicator_3-2-2.csv

The command "ruby ./scripts/csvcheck.rb" exited with 1.
```

The last line is in red.

**Comment by AlexLewis1** [comment]
I didn't realise the error came along with that line, which makes it really easy to spot the error. Thank you

File: data/indicator_3-2-2.csv [hunk 1]
```diff
@@ -1,152 +1,531 @@
...
+2009,Wales,,,"Betsi Cadwaladr University ",,,,4
+2010,Wales,,,"Betsi Cadwaladr University ",,,,3.5
+2011,Wales,,,"Betsi Cadwaladr University ",,,,3.2
+2012,Wales,,,"Betsi Cadwaladr University ",,,,3.5
+2013,Wales,,,"Betsi Cadwaladr University ",,,,1.9
+2014,Wales,,,"Betsi Cadwaladr University ",,,,2.3
+2015,Wales,,,"Betsi Cadwaladr University ",,,,2.7
+2009,Wales,,,"Powys Teaching ",,,,2.4
+2010,Wales,,,"Powys Teaching ",,,,4.1
+2011,Wales,,,"Powys Teaching ",,,,:
```
Comment (dougmet) [inline data/indicator_3-2-2.csv]: Need to remove this colon

**Comment by dougmet** [comment]
I wouldn't go so far as to say really easy! But we can work with it. Turns out there are two issues. The code doesn't handle missing values properly (I'm fixing that now) but there is also a colon on line 283 in the csv which needs to be removed.

This is all really useful testing

**Comment by dougmet** [comment]
This was completed in #77

**Pull request closed without merging**